android/src/main/java/com/polidea/reactnativeble/BleModule.java

```diff
@@ -259,6 +267,9 @@ public void startDeviceScan(@Nullable ReadableArray filteredUUIDs, @Nullable Rea
     }
 
     private void safeStartDeviceScan(final UUID[] uuids) {
+        if (rxBleClient == null) {
+            throw new IllegalStateException("Ble client not created when tried to start device scan");
```

Comment:
Use JS names if possible. I think that way user will be able to understand it better. Ble client -> BleManager